venues/ICLR.cc/2018/Conference/python/programchairs/get-accepted.py

```diff
@@ -26,31 +26,34 @@ def load_decisions(client):
             dec_info[decision.forum] = decision.content['decision']
     return dec_info
 
+def load_profile(profile_info, author, profile):
```

Comment:
these changes are already in master, you need to sync your fork first:

https://help.github.com/articles/syncing-a-fork/

and update your branch with the latest changes of master

Reply:
https://github.com/iesl/openreview-scripts/blob/master/venues/ICLR.cc/2018/Conference/python/programchairs/get-accepted.py#L29